scripts/justmat/foulplay.lua

```diff
@@ -143,6 +218,20 @@ function init()
   else
     clk:start()
   end  
+  
+  -- set up grid
+  print("grid")
+  -- grid refresh timer, 40 fps
+  -- this caught me out first time 
+  -- the template just updates the grid from keys 
+  -- but it seems better to treat the grid like the screen
+  -- also g: isn't set yet during init 
```

Comment:
FYI @tehn.  

i think we talked about how it's confusing that `g` is unavailable in `init`.  (can't remember where though.)

anyway, this might be another thing to document in "the life of a script" (https://github.com/monome/norns/issues/437) or tackle in the big callback rethink (https://github.com/monome/norns/issues/444).